From f8fc07787591af5f4ccd1ed24cdef30b5871c799 Mon Sep 17 00:00:00 2001
From: Kyle Lape <klape@redhat.com>
Date: Mon, 23 Feb 2018 13:05:00 -0600
Subject: [PATCH] Bump release to 7

---
 insights_client/constants.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/insights_client/constants.py b/insights_client/constants.py
index 4e77354..4784e67 100644
--- a/insights_client/constants.py
+++ b/insights_client/constants.py
@@ -5,4 +5,4 @@ Constants
 
 class InsightsConstants(object):
     app_name = 'insights-client'
-    version = '3.0.3-7'
+    version = '3.0.3-8'
-- 
2.14.3

